app/src/main/java/com/example/parkingsystemkotlin/mvp/model/reservation/ReservationInformationDB.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.parkingsystemkotlin.mvp.model.reservation
+
+object ReservationInformationDB : DataBase {
+
+    private var reservationInfomation: MutableMap<String, MutableList<Reservation>>
+
+    init {
+        reservationInfomation = mutableMapOf()
+    }
```

Comment:
this its ok, but if you are created a mutableMap in the initialisation state you could instantiate in the definition

Reply:
done